### ISSUE TITLE
add headers to WHEP server calls

### ISSUE THREAD

**Issue by hakim89** [Eyevinn/webrtc-player#66]
I have a media server serving multiple streams over whep protocol but requires Basic Auth. 
Any way I can add headers to all server calls WebRTCPlayer makes?
Thank you

**Comment by friday** [comment]
It sounds like something we should be able to support. We will try to look into this soon.

**Comment by hakim89** [comment]
Thank you. 
I did create my own adapter and set the authKey with the basic auth header and it worked.
I can also see how the load function in main branch has a new arg as the authKey (not in the 0.12.1 tag) so hopefully this is coming in newer versions. 
```
 async load(channelUrl: URL, authKey: string | undefined = undefined) {
    this.channelUrl = channelUrl;
    this.authKey = authKey;
    this.connect();
  }
```
Thank you

**Comment by friday** [comment]
Looks like you are :100: correct. The authKey is used as the `Authorization` header, so this should already work for the WHEP adapter (only), but we haven't released it. 

```ts
const basicAuthKey = btoa(`${user}:${passwd}`);
const authKey = `Basic ${basicAuthKey}`;
await player.load(new URL(channelUrl), authKey);
```

Do you want to make a release @birme or should I?

**Comment by birme** [comment]
Cool, I can fix a release!

**Comment by birme** [comment]
A new release including this feature is now available (v0.13.0)
